commands/upload/upload.go

```diff
@@ -530,7 +526,7 @@ func determineBuildPathAndSketchName(importFile, importDir string, sk *sketch.Sk
 	// Case 1: importFile flag has been specified
 	if importFile != "" {
 		if importDir != "" {
-			return nil, "", fmt.Errorf(fmt.Sprintf(tr("%s and %s cannot be used together"), "importFile", "importDir"))
+			return nil, "", fmt.Errorf(tr("%[1]s and %[2]s cannot be used together", "importFile", "importDir"))
```

Comment:
```suggestion
			return nil, "", fmt.Errorf(tr("%s and %s cannot be used together", "importFile", "importDir"))
```
See explanation above.